docs/installation/setup_configuration.rst

```diff
@@ -0,0 +1,64 @@
+.. _installation_configuration_cli:
+
+==============================
+Open Klant configuration (CLI)
+==============================
+
+After deploying Open Klant, it needs to be configured to be fully functional.
+The django management command ``setup_configuration`` assist with this configuration.
+You can get the full command documentation with:
+
+.. code-block:: bash
+
+    ./src/manage.py setup_configuration --help
```

Comment:
```suggestion
    python ./src/manage.py setup_configuration --help
```
There is no (manual) install documentation for OpenKlant so there is no standard. But the python command is included in open zaak and the default projects.

Same with the command below

Reply:
Note that this is not required (as there is a shebang in the `manage.py`) but for consistency I added them :+1: